lib/DBus/Client.hs

```diff
@@ -125,27 +153,42 @@ module DBus.Client
     , clientThreadRunner
     , defaultClientOptions
     , connectWith
+
+    , errorInvalidParameters
     ) where
 
 import Control.Concurrent
+
+import Control.Applicative
+import Control.Arrow
+import qualified Control.Exception
 import Control.Exception (SomeException, throwIO)
-import Control.Monad (forever, forM_, when)
+import Control.Lens hiding (coerce)
```

Comment:
`Module ‘Control.Lens’ does not export ‘coerce’`

Reply:
It did in one of the older versions of lens. Without this one of the travis builds fails.

Reply:
Probably one you removed already. Just checked, all builds have this warning.